res/database/Default/database_news.xml

```diff
@@ -2668,7 +2668,7 @@ Demonstrations are expected again this year.</en>
 		<news id="bc7adf97-1b60-4745-99b3-f18604837103" thread_id="f0228a97-bf9b-4d35-9723-9dcbbbc99b1b" type="0" creator="">
 			<title>
 				<de>Stunthund bei Dreharbeiten verletzt</de>
-				<en>Stunt dog injured during shooting</en>
+				<en>Stunt dog injured during filming</en>
```

Comment:
But injuries during shooting would make more sense ;-)

Reply:
yeah but I guess that wasn't the intention